packages/at_persistence_secondary_server/lib/src/model/at_meta_data.dart

```diff
@@ -245,7 +252,9 @@ class AtMetaData extends HiveObject {
           encAlgo == other.encAlgo &&
           ivNonce == other.ivNonce &&
           skeEncKeyName == other.skeEncKeyName &&
-          skeEncAlgo == other.skeEncAlgo;
+          skeEncAlgo == other.skeEncAlgo &&
+          pubKeyHash == other.pubKeyHash &&
+          immutable == other.immutable;
```

Comment:
note that pubKeyHash had not been previously added to the `==` or hashCode operations, a bug which would have caused issues in some situations. Added it as well as the new immutable flag